docs/src/pages/components/cdk-components/text-input/api.jsx

```diff
@@ -49,7 +49,9 @@ const textInputPropsTable = () => {
           Action to be shown in the input. This is an object composed of an
           onClick function and an icon, being the latter either an inline svg or
           a URL to the image. An example of this object is: {"{ "}
-          <code>onClick: onClickFunction, icon: string | svgIcon</code>
+          <code>
+            onClick: onClickFunction, icon: string | svgIcon, title: string
```

Comment:
For consistency, I think it would be better to use `function` instead of `onClickFunction`, as it is its type in javascript and the one used in the rest of the props of this kind.

Reply:
Done.